classes/Device.class.php

```diff
@@ -1871,7 +1871,7 @@ function GetChildDevicePicture($parentDetails, $rear=false){
 						$slot->H=$ymax-$ymin;
 					}else{
 						// Last slot isn't defined so just error out
-						break;
+						return;
```

Comment:
Yeah, I just figured you can only break out of loops (this isn't in a loop.) PHP was complaining about it: PHP Fatal error:  'break' not in the 'loop' or 'switch' context in classes/Device.class.php on line 1875